runtime/doc/options.txt

```diff
@@ -2299,7 +2299,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	directory.  If you switch this option on you should also consider
 	setting the 'secure' option (see |initialization|).  Using a local
 	.exrc, .nvimrc or .ngvimrc is a potential security leak, use with care!
```

Comment:
I don't want to bog down this [fairly simple] PR, so I guess just removing the `ngvimrc` case in `ex_cmds2.c` should be done, as I'm not sure about the other cases.